app/src/@types/Settings.d.ts

```diff
@@ -6,8 +6,9 @@ interface Settings {
     vimcEmail: string;
     slackUrl: string;
     isApplicantTouchstone: (id: string) => boolean;
-    isStochasticTouchstone: (id: string) => boolean;
     is2017Touchstone: (id: string) => boolean;
+    isVersionOfStochasticTouchstone: (touchstoneId: string) => boolean;
```

Comment:
Named this way partly to distinguish from old isStochasticTouchstone method, which should now be obsolete